access-control.yaml

```diff
@@ -347,10 +347,9 @@ teams:
   - name: besu-contributors
     maintainers:
       - garyschulte
-      - jframe
```

Comment:
why did you remove me from the besu-contributors?

Reply:
only so there was one member in the group but I think it's fine for the group to be only maintainers

Reply:
so I reverted this change to your membership :)